owasp-security-logging-log4j/pom.xml

```diff
@@ -31,8 +31,19 @@
     <url>https://github.com/javabeanz/owasp-security-logging</url>
   </scm>
   <properties>
-    <log4j.version>2.16.0</log4j.version>
+    <log4j2.version>2.16.0</log4j2.version>
```

Comment:
I updated this project to use the `log4j-bom` file for slightly easy dependency management.  I debated pushing it to the parent but wanted to take my time :)

Reply:
The parent pom should not need it -we don't want to force logback users to import Log4J.